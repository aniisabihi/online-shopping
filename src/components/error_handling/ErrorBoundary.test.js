import React from "react";
import { shallow } from "enzyme";
import ErrorBoundary from "./ErrorBoundary";

const ErrorBoundaryComponent = <ErrorBoundary />;

let errorBoundary = shallow(ErrorBoundaryComponent);

/* RENDER COMPONENT */
describe("Render ErrorBoundary", () => {
  it("should render the ErrorBoundary Component accordingly", () => {
    expect(errorBoundary).toMatchSnapshot();
  });
});
