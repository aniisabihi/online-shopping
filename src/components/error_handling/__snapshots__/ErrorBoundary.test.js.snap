// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ErrorBoundary should render the ErrorBoundary Component accordingly 1`] = `ShallowWrapper {}`;
